docs/troubleshooting.asciidoc

```diff
@@ -31,6 +31,23 @@ In order to debug the agent consider performing the following steps:
 3. Wait at least 5 seconds after the page has loaded
 4. Monitor the Console and Network panel in your browsers developer tools
 
+[float]
+[[disable-events-payload-compression]]
+=== Disable events payload compression
+
+In browsers such as Chrome the payload of the events sent by the RUM JS agent is being compressed with gzip.
+Because of that, you will not see a readable content when inspecting the event with the Network panel of your browser developer tools.
```

Comment:
```suggestion
In browsers such as Chrome, the RUM agent event payload is compressed with gzip.
Because of this compression, you will not see readable content when inspecting the event with the Network panel of your browser developer tools.
```